rockcraft.yaml

```diff
@@ -0,0 +1,75 @@
+name: spark
+summary: Spark ROCK
+description: Spark ROCK
+license: Apache-2.0
+
+version: "0.1"
+base: ubuntu:20.04
+
+entrypoint: ["/opt/entrypoint"]
```

Comment:
`/opt` seems like a weird location for an entry point - it should be probably just be pebble anyway :)

Reply:
Ack